.envrc

```diff
@@ -159,57 +141,50 @@ done
 
 ### Git ###
 
-info "Configuring git..."
+debug "Configuring git..."
 
 make setup-git-config
 
 ### Python ###
 
 venv_name=".venv"
 
-info "Activating virtualenv..."
-
 if [ ! -f "${venv_name}/bin/activate" ]; then
-    info "You don't seem to have a virtualenv."
-    # If the version doesn't match the contents of .python-version it will fail
-    [[ $(pyenv version) ]] ||
-        die "pyenv does not have the right Python version installed. Please run:
-    make setup-pyenv && python3.6 -m venv ${venv_name}"
-    advice_init_venv "$venv_name"
+    # This is time consuming but it has to be done
+    source ./scripts/bootstrap-py3-venv
```

Comment:
This is a big change but it only happens when we have no `.venv`.

Reply:
I would re-word it to be more assertive: `You are missing a Python virtualenv and we *need* to run a bootstrapping script (it can take a few minutes)`

Reply:
I've updated the wording.